### PR TITLE
added new files for each pages

### PR DESCRIPTION
**Pull request by ankitjain0105**
Added files as per Sounds pages, Please approve.

### CODE REVIEW

File: config/sounds-web/all.js [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
+
 module.exports = {
-  options: {
-    hide: ['orb', 'bbccookies-prompt', 'smphtml5iframebbcMediaPlayer', 'smphtml5iframesmp-wrapper', 'edr_l_first', 'bbcprivacy-prompt', 'id4-cta-', 'msi-modal', 'p_audioui_'],
-    skip: []
-  },
+  options,
   baseUrl,
   paths: [
     '/sounds',
```
Comment (andymsuk) [inline config/sounds-web/all.js]: Think you could update this to pull in the paths from the other config files to save defining the paths in two places.

Comment (ankitjain0105) [reply config/sounds-web/all.js]: Done, Please have a look.

File: config/sounds-web/playspace.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+module.exports = {
+  options,
+  baseUrl,
+  paths: [
+    '/sounds/play/m0009353',
+    'sounds/play/live:bbc_radio_one'
+  ],
+  signedInPaths: [
+    '/sounds/play/m0009353',
+    'sounds/play/live:bbc_radio_one'
```
Comment (andymsuk) [inline config/sounds-web/playspace.js]:
```suggestion
    '/sounds/play/live:bbc_radio_one'
```

**Comment by andymsuk** [comment]
Closing in favour of #33

**Pull request closed without merging**